coops-ui/src/views/AnnualReport.vue

```diff
@@ -153,6 +153,37 @@
                 <Directors @directorsChange="directorsChange" ref="directorsList" :asOfDate="agmDate" />
               <!-- </v-card> -->
             </section>
+
+            <!--Certify -->
+            <section>
+              <header>
+                <h2 id="AR-step-4-header">4. Certify Correct</h2>
+                <p>Enter the name of the current director, officer, or lawyer submitting this Annual Report.</p>
+              </header>
+              <v-card flat id="AR-step-4-container">
```

Comment:
This block (and its functionality below) should be in a separate sub-component.

Reply:
I concur, this should be in its own component.